lib/ui/platform_dispatcher.dart

```diff
@@ -314,15 +314,8 @@ class PlatformDispatcher {
   /// callback is invoked indicating the new view that has focus and the direction
   /// in which focus was received. For example, if focus is moved to the [FlutterView]
   /// with ID 2 in the forward direction (could be the result of pressing tab)
-  /// the callback receives the following [ViewFocusEvent]:
-  ///
-  /// ```dart
-  /// ViewFocusEvent(
```

Comment:
For analyzing the snippets, dart:ui is imported as `import 'dart:ui' as ui`. So, you need to prefix everything coming from `dart:ui` with `ui.` in these samples.

https://github.com/flutter/flutter/blob/b209125d11affe911da7f9694aad78f87f1453d2/dev/bots/analyze_snippet_code.dart#L506

Reply:
Interesting, these patterns are somewhat hidden. It is not intuitive what's expected.

Also, why don't we make these verifications as part of the engine submit process?